docs/source/using/experimental-features.adoc

```diff
@@ -73,6 +73,38 @@ If you prefer to use dynamic assignment, you can use:
 $ minishift ip --set-dhcp
 ----
 
+[[local-proxy-server]]
+== Local Proxy Server
+
+{project} provides an option to run a local proxy server on the host, which the {project} instance can use to access the internet.
```

Comment:
s/provides an option to run a local proxy server on the host/Runs a local proxy server on the host/. 

Because when user will run the feature it is not an option, so we should say what it does explicitly.

Reply:
Suggestion: `Runs a local proxy server on the host. {project} instance can use this proxy for all external network traffic`.